src/vstest.console/Publisher/IMetricsPublisher.cs

```diff
@@ -13,8 +13,13 @@ public interface IMetricsPublisher
         /// <summary>
         /// Publish the Metrics
         /// </summary>
-        /// <param name="eventName"></param>
-        /// <param name="metrics"></param>
+        /// <param name="eventName">The event Name</param>
+        /// <param name="metrics">Key/Value pair of Properties and Values</param>
         void PublishMetrics(string eventName, IDictionary<string, string> metrics);
+
+        /// <summary>
+        /// Dispose the Telemetry Session
+        /// </summary>
+        void Dispose();
```

Comment:
Extend your interface from IDosposable. No need to have Dispose()

Reply:
done